### ISSUE TITLE
Do not trigger SlevomatCodingStandard.ControlStructures.EarlyExit.EarlyExitNotUsed when there is only 1 line inside

### ISSUE THREAD

**Issue by badraxas** [slevomat/coding-standard#1691]
Hello, this sniff always tells me to add an early exit even if there is only one line in the if block.

For example:
```php
foreach ($array as $word) {
    if ($word === 'hello') {
        $greeting[] = $word;
    }
}
```

The sniff suggests modifying the code to:
```php
foreach ($array as $word) {
    if ($word !== 'hello') {
        continue;
    }

    $greeting[] = $word;
}
```

In my opinion, this doesn't reduce code nesting and just creates unnecessary complexity. It's justified only when we have more than one line inside, like this:

```php
foreach ($array as $word) {
    if ($word === 'hello') {
        $greetings[] = [
            'hello',
            'bonjour',
            'hola',
            'hallo',
            'hej',
            'привет',
        ];
    }
}
```

My suggestion is to ignore blocks with only one line (or even better, add a parameter to let the user choose their own limit).

**Comment by badraxas** [comment]
Sorry, I'm not aware of `ignoreStandaloneIfInScope` and `ignoreOneLineTrailingIf`
